### PR TITLE
Home page overview section #4

### PR DESCRIPTION
**Pull request by mumbi58**
# Description
Summary of Changes:
I have built an overview page using HTML and styled it using CSS to create a visually appealing and informative web page.
Motivation: To help many people learn easily and enjoyably, Elewa uses a people-focused approach to make education and training accessible for all.

Dependencies:
There are no external dependencies for this change. It was implemented using standard HTML and CSS techniques, making it compatible with a wide range of web environments.
#4 
## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Screenshot 
large screens view
image
 phone view below
image
tablet view below
image

## How Has This Been Tested?
I used manual cross-device testing which involves manually resizing the browser window or using the browser's built-in device emulation features to simulate how your website looks and behaves on various devices such as mobile phones, tablets, and desktops.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  transform: translateY(25%);
+  font-size: 22px;
+  margin-right: 5%;
+}
+
+/* Image styles */
+.pic img {
+  max-width: 55%;
+  margin-left: 35%;
+  margin-top: 10%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use shorthand

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  .subcontainer {
+    flex: 2;
+    transform: translateY(25%);
+    font-size: 12px;
+    margin-right: 10%;
+  }
+  .pic img {
+    max-width: 55%;
+    margin-left: 15%;
+    margin-top: 5%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here also

File: src/css/style.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  .container {
+    display: block;
+    width: auto;
+  }
+  .subcontainer {
+    flex: 2;
+    transform: translateY(25%);
+    font-size: 18px;
+    margin-right: 10%;
+    margin-left: 10%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

File: src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  }
+  .pic img {
+    max-width: 55%;
+    margin-left: 15%;
+    margin-top: 5%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: here

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Very good attempt, the mobile screen align alignment seems a bit off, anf the font weight, style and sizes. With these changes we'll be good to go

**Comment by mumbi58** [comment]
i have made the changes

File: src/css/style.css (hunk 4 above)
Comment (mumbi58) [reply src/css/style.css]: i have made the changes requested

**Comment by JAPHETHNYARANGA** [comment]
Good attempt but this issue has been closed . Please pick another open issue. Hint; don't delete code already in a file when working on it.

**Pull request closed without merging**